blocks/default-output-block/output-types/__tests__/default.test.jsx

```diff
@@ -272,7 +327,10 @@ describe('chartbeat render conditions', () => {
     }))));
 
     const { default: DefaultOutputType } = require('../default');
-    const wrapper = shallow(<DefaultOutputType deployment={jest.fn()} metaValue={jest.fn().mockReturnValue('article')} />);
+    const wrapper = shallow(
+      // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I believe we have an ignore for this rule globally in the eslint config. if not, can you add one. we use props spreading all over the place and aren't against this as a team